releasenotes/notes/print-vibrational-excited-state-energies-0ea51457b3a4a69f.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    Fixes the formatting of the occupied modals and adds the excited state energies
+    when printing a :class:`~qiskit_nature.results.VibrationalStructureResult`.
+  - |
```

Comment:
I'm not sure whether this is needed. @woodsp-ibm @manoelmarques wdyt?

Reply:
You mean the release note? Since this fixes a problem that an end user would encounter when/if printing the result we need a release note in this case I think.

Reply:
No, I actually specifically meant this additional line containing the `- |`. Is that needed for some formatting? I have never seen us use it. Instead we usually just have an empty line and then a new paragraph.

Reply:
I saw the `- |` also in the other reno files, so I assumed that it is needed.

Reply:
I don't see it being used between paragraphs anywhere else. I'm not sure whether it really matters though. @manoelmarques do you have an opinion on this?

Reply:
This syntax is there to specify multiple release notes. In reno each list entry in a release note file indicates a new release note in that section. So if you look at the rendered output you'll see 2 release notes (ie 2 bullet points) under the fixes section. If that is the intent here to have 2 distinct notes than this is correct, if you want 1 combined note you can remove this and just have a new line. Based on my reading I assume the intent is to have a single release note and just a new paragraph so I would remove it.

The exact yaml syntax being used here is `-` indicates a new list entry (at a single indent means under the fixes key) while `|` is there to indicate preserve newlines for this block (at the same indentation level for the list entry block).